09_proxmox_fstrim_cloudinit_ansible_lvm_08-sep-2022/fstrim_timer.md

```diff
@@ -14,15 +14,24 @@ According to official documentation, we need
 -   Enable `Discard` and `SSD Emulation` flags for drive
```

Comment:
could you please attach a screenshot (or make a reference to another lesson with this screenshot)?